src/irc.lua

```diff
@@ -15,6 +15,21 @@ irc.conn = function (irc_config)
     irc.connection:send(('USER %s * 8 :%s\r\n'):format(irc_config.ident, irc_config.gecos))
 end
 
+irc.action = function (target, msg)
+    irc.connection:send(('PRIVMSG %s :\x01ACTION %s\x01\r\n'):format(target, msg))
+end
+
+irc.CTCP_query = function (client, query)
```

Comment:
no other function has all-caps despite all of the associated verbs being all-caps'd. I think this should probably just be ``ctcp()`` rather than ``CTCP_query()``.

Reply:
The caps were because ctcp is an acronym and I was using proper english XD, but yeah, not important.

As for the name having query, there are also ctcp replies, so I wanted to make sure queries would be differentiated from the replies function that I was going to write in the future. Did you want to have all event reply functions be called "react_to_..." ? then the actual bot commands just be the name of the command? I am fine with whatever convention.

Reply:
the reaction functions are actually likely to be in a separate file (briefly mentioned out-of-band), so I would not worry about confusion with those names. please do rename it :)